Events/Event.swift

```diff
@@ -8,6 +8,9 @@
 
 import Foundation
 
+/**
+* Event submodule contains implementation of memory leak-less listeners using closures
```

Comment:
Not really `leak-less`. It all depends on the user, because he needs to make sure not to create a strong retain cycle with the closures.